README.md

```diff
@@ -158,11 +158,11 @@ Example Playbook
             - name: shared
               size: "100 GiB"
               mount_point: "/mnt/app/shared"
-              #fs_type: xfs
+              #fs_type: 'xfs'
               state: present
             - name: users
               size: "400g"
-              fs_type: ext4
+              fs_type: 'ext4'
```

Comment:
This string does not need to be quoted.  What coding style are you applying here?  That is, where does it say that this string should be quoted?

Reply:
I just want to keep fs_type consistent,  I'm updating it because I found some of the code quoted and some don't have.
If this kind of change is not necessary, I'm fine to keep it.

Reply:
> I just want to keep fs_type consistent, I'm updating it because I found some of the code quoted and some don't have.
> If this kind of change is not necessary, I'm fine to keep it.

I think it is not necessary.

Reply:
OK, thanks for your review.